js/dataframe/styles.scss

```diff
@@ -4,7 +4,7 @@
   box-sizing: border-box;
 }
 
-.shiny-data-grid {
+shiny-data-frame {
```

Comment:
Change to `shiny-data-frame .shiny-data-grid {`

Move the new variables into `shiny-data-frame {`

Or at least make the selectors consistent.